hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsRestOperationType.java

```diff
@@ -42,4 +42,19 @@ public enum AbfsRestOperationType {
     DeletePath,
     CheckAccess,
     LeasePath,
+    CreateContainer,
```

Comment:
Here also, hw about grouping the enums ?
Global enum
dfs enum
blob enum

Reply:
This sounds good from code readability POV.
But what I get from this is that you are suggesting instead of having a single enum class name `AbfsRestOperationType`, we introduce three enums, one for each category like `AbfsGlobalRestOperationType`, `AbfsDFSRestOperationType` and `AbfsBlobRestOperationType`.
Please let me know if I am getting it wrong.
But, what i feel is doing so will lead to a lot of code change at every place these are used. I thin  we can take it up as a separate work item and try to minimize code changes in these PRs bringing in new support.

What do you suggest??

Reply:
ok, please create another follow-up PR as this change itself is quite big.

Reply:
Follow up task: https://issues.apache.org/jira/browse/HADOOP-19333